app/src/main/java/tttTDD/Java/ConsoleClient.java

```diff
@@ -144,13 +144,26 @@ private String formatCellDisplay(String cell, String maxInput) {
         StringBuilder formattedCell = new StringBuilder();
 
         if (cell.length() <= maxInput.length()) {
-            StringBuilder spaceAlignment = new StringBuilder();
 
-            IntStream.range(0, maxInput.length() - cell.length()).forEach(i -> spaceAlignment.append(" "));
-            formattedCell.append(spaceAlignment);
+            IntStream.range(0, maxInput.length() - cell.length()).forEach(i -> formattedCell.append(" "));
             formattedCell.append(cell);
         }
 
         return formattedCell.toString();
     }
+
+    private int playerInput() {
+        int value;
+
+        try {
+            value = input.nextInt();
+        }
+        catch (Exception e) {
+            System.out.println("Stop it. Enter a number as an input:");
+            input.nextLine();
+            value = playerInput();
```

Comment:
What happens if you just `return playerInput();` here?

Reply:
it does the same thing lol. The reason behind using a variable was more for readability purposes.

Reply:
I think for recursion like this, it kind of added to my confusion. By assigning a variable, at first I thought the code flow would continue on. 

In general, I think having an explicit return here would more clearly indicate the end of the execution path for this function / call stack, when an exception occurs